indico/htdocs/js/indico/modules/eventdisplay.js

```diff
@@ -1,74 +1,55 @@
-$(document).ready(function() {
-    'use strict';
+/* This file is part of Indico.
+ * Copyright (C) 2002 - 2016 European Organization for Nuclear Research (CERN).
+ *
+ * Indico is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation; either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * Indico is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Indico; if not, see <http://www.gnu.org/licenses/>.
+ */
 
-    $('.event-service-row > .trigger').on('click', function() {
-        var toggler = $(this);
-        toggler.siblings('.event-service-details').slideToggle({
-            start: function() {
-                toggler.toggleClass('icon-expand icon-collapse');
-            },
-            duration: 'fast'
-        });
-    });
+(function(global) {
+    'use strict';
 
-    $('.event-service-row-toggle').on('click', function(e) {
-        e.preventDefault();
-        var toggler = $(this);
-        var toggler_button = $(this).parent().siblings('.trigger');
-        toggler.parent().siblings('.event-service-details').slideToggle({
-            start: function() {
-                toggler_button.toggleClass('icon-expand icon-collapse');
-            },
-            duration: 'fast'
+    global.setupEventDisplay = function setupEventDisplay() {
+        $('.js-go-to-day').dropdown({
+            always_listen: true
+        }).find('li a').on('menu_select', function() {
+            window.location = $(this).attr('href');
```

Comment:
`location.href = ...` (matter of preference though, I just find it weird to assign a string to what usually holds an object)